gpcontrib/arenadata_toolkit/arenadata_toolkit--1.0--1.1.sql

```diff
@@ -49,3 +49,131 @@ JOIN pg_catalog.pg_class pgc ON (skew.skewoid = pgc.oid)
 JOIN pg_catalog.pg_namespace pgn ON (pgc.relnamespace = pgn.oid);
 
 GRANT SELECT ON arenadata_toolkit.adb_skew_coefficients TO public;
+
+/*
+ This is part of arenadata_toolkit API for ADB Bundle.
+ This function collects information in db_files_current and db_files_history tables.
+ */
+CREATE OR REPLACE FUNCTION arenadata_toolkit.adb_collect_table_stats()
```

Comment:
What should I do if I already installed version 1.1 with a previous version of this function?

Reply:
Version 1.1 is auto-generated from "1.0" and 1.0--1.1" scripts:
```
DATA = \
       arenadata_toolkit--1.0.sql \
       arenadata_toolkit--1.0--1.1.sql

DATA_built = $(EXTENSION)--$(EXTENSION_VERSION).sql

$(DATA_built): $(DATA)
	cat $(DATA) > $(DATA_built)
```

Reply:
> Version 1.1 is auto-generated from "1.0" and 1.0--1.1" scripts:

Yes, but before this patch, the `arenadata_toolkit--1.0--1.1.sql` file already contained the `adb_collect_table_stats` function (which got there from the `arenadata_toolkit--1.0.sql` file), which means that before the patch I already had this function in version 1.1 and it will not update!

Reply:
Try installing version 1.1 before the patch. How will the patch update the function later?

Reply:
Moved to version 1.2